code/game/objects/items/devices/instruments.dm

```diff
@@ -79,6 +86,34 @@
 		return
 	return changeInstrument(chosen)
 
+/obj/item/instrument/piano_synth/headphones
+	name = "headphones"
+	desc = "Unce unce unce unce. Boop!"
+	icon = 'icons/obj/clothing/accessories.dmi'
+	lefthand_file = 'icons/mob/inhands/clothing_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/clothing_righthand.dmi'
+	icon_state = "headphones"
+	item_state = "headphones"
+	slot_flags = ITEM_SLOT_EARS | ITEM_SLOT_HEAD
+	force = 0
+	w_class = WEIGHT_CLASS_SMALL
+	custom_price = 125
+	instrumentRange = 1

```

Comment:
Shouldn't this be zero ? Unless you want nearby people to hear it too.

Reply:
Yes it's intentional, wanted to simulate how you can hear someone's headphones playing if you're right next to them

Reply:
Alright